models/Document/Tag/Link.php

```diff
@@ -115,14 +115,7 @@ public function frontend()
 
             if (isset($this->options['noText']) && $this->options['noText'] == true) {
                 $noText = true;
-            }
-
-            // add attributes to link
-            $attribs = [];
-            foreach ($this->options as $key => $value) {
-                if (is_string($value) || is_numeric($value)) {
-                    $attribs[] = $key.'="'.$value.'"';
-                }
+                unset($this->options['noText']);
```

Comment:
@brusch just a heads up...this change will ignore attributes if any used except what is mentioned here https://pimcore.com/docs/5.x/Development_Documentation/Documents/Editables/Link.html#page_Configuration